.clang-tidy

```diff
@@ -88,7 +93,17 @@ hicpp-*,
 #
 # * rule: readability-use-anyofallof
 #   justification: requires C++20 and std::ranges but we only use C++14
-
+#
+# * rule: cppcoreguidelines-non-private-member-variables-in-classes
+#   justification: Sometimes it makes sense to have protected members to extend the base class.
+#                  Even public members are useful in lightweight classes (effectively structs).
+#                  This unnecessarily limits design options.
+#
+# * rule: readability-named-parameter
+#   justicfication: For tag types which just guide overloading no name is needed.
+#                   Declaration of private functions does not require a name (no doxygen comment).
+#                   For callbacks omitting the name is less verbose than to cast an unsued parameter to void.
+#
```

Comment:
Could you please add a small example which shows the violation and in the `readability-named-parameter` how we want it, e.g.
```
bad
  auto callable = [] (auto /*foo*/) {...};
good
  auto callable = [] (auto) {};
```